woocommerce/woocommerce-ios
# Add crowdsignal survey URL rule

Description
==========
Fixes issue #3006. Our survey implementation code as it is right now is not submitting in any way a clear platform identification to achieve an easy identification of the user inputs on our surveys.

How to Test
==========

1. Go to the My Store section
2. Click on the Settings icon on the upper right corner of the toolbar
3. Once the settings view is presented, click on the `Send feedback` option
4. Wait for the survey loading and make sure everything is working the same way as before
5. Complete the Survey Form
6. Go to the CrowdSignal dashboard with a valid A8C account and go to the respective Survey overview page you just answered
7. Click at the far right corner of the Overview page and click on the export button to download the Survey report, you can export in both CSV or Excel, it's up to your choice.
8. Go to the right end of the report file and verify if the custom tags are presented according to the Survey you just answered

Update release notes:
